spec/index.html

```diff
@@ -1802,42 +1814,59 @@ <h3>RDF Triples Constructors</h3>
         |curPredicate| <var>N</var> .</span>
     </p>
 
-    <h4 id="tripleTerm" style="padding-bottom:0; margin-bottom:0;"><span>Triple Terms:</span></h4>
+    <h4 id="tripleTerm" style="padding-bottom:0; margin-bottom:0;"><span>Triple Terms</span></h4>
 
     <p>Beginning the <a href="#grammar-production-tripleTerm"><code>tripleTerm</code></a> production
       records the |curSubject| and |curPredicate|.
       Finishing the <a href="#grammar-production-tripleTerm"><code>tripleTerm</code></a> production
-      and restores the recorded values of |curSubject| and |curPredicate|.</p>
+      restores the recorded values of |curSubject| and |curPredicate|.</p>
+
+    <h4 id="reifier" style="padding-bottom:0; margin-bottom:0;"><span>Reifier</span></h4>
+
+    <p>Beginning the <a href="#grammar-production-reifier"><code>reifier</code></a> production,
+      if |curReifier| is set, the production yields the RDF triple |curReifier| <code>rdf:reifies</code> |curTripleTerm|.
+      Finishing the <a href="#grammar-production-reifier"><code>reifier</code></a> production,
+      |curReifier| is taken from the <a href="#handle-reifier"><code>reifier</code></a> term constructor.</p>
```

Comment:
Delete this text. Moved to earlier.

Reply:
The reason it is written the way it is is that `Reifier` can set the value that might be used in `Annotation Block`, but we'll only know that if there is an Annotation Block after the Reifier. If it was set, but not used, and we see another Reifier, it can then be emitted for the saved `curReifier`.